### PR TITLE
updated styling to header component and added relavent react componen…

### PR DESCRIPTION
**Pull request by Denise-Carlson**
…t imports in header

### CODE REVIEW

File: app/javascript/components/components/Header.js [hunk 1]
```diff
@@ -14,9 +14,22 @@ class Header extends Component {
...
+  >
+    <NavbarBrand href="/">
+      JudoTen
+    </NavbarBrand>
+    <NavbarToggler onClick={function noRefCheck(){}} />
+    <Collapse navbar>
+      <Nav
+        className="me-auto"
+        navbar
+      >
```
Comment (WalkerTexas-coder) [inline app/javascript/components/components/Header.js]: Indentation, feel free to use a code formatter

**Review by WalkerTexas-coder — COMMENTED** [review]
Indentation

File: app/javascript/components/components/Header.js (hunk 1 above)
Comment (Denise-Carlson) [reply app/javascript/components/components/Header.js]: made required changes to indentation

**Review by Denise-Carlson — COMMENTED** [review]
changes look good. Replying to Austin.

**Review by WalkerTexas-coder — APPROVED** [review]
Let's Go!

**Pull request merged**